components/registration/registration-controller.js

```diff
@@ -658,6 +658,7 @@ trackerCapture.controller('RegistrationController',
             $scope.selectedTei.orgUnit = $scope.tei.orgUnit = $scope.selectedOrgUnit.id;
             $scope.selectedTei.attributes = $scope.tei.attributes = [];
         }
+        $scope.tei.featureType = $scope.selectedTei.featureType;
         $scope.tei.geometry = $scope.selectedTei.geometry;

```

Comment:
Geometry is the old name for feature type... perhaps this one can be removed?

Reply:
This geometry object looks like this: 
geometry: {
    type:  A leaflet object type ("Polygon" in this case)
    coordinates: The vertices of the polygon.
}

while `tei.featureType` is a string that should contain the same value as `geometry.type`. Thus it seems more likely that it's the other way around, that `featureType` is the old name for `geometry`. However, the server still uses `featureType` as condition for deleting existing polygons, so it seems both are needed.